backend/CMakeLists.txt

```diff
@@ -30,7 +30,8 @@ set(TFL_BUILD_DIR ${TF_ENV}/tflite_build)
 set(TFL_LIB_PATH /usr/lib/libtensorflowlite.so)
 set(TFL_GPU_LIB_PATH /usr/lib/libtensorflowlite_gpu_delegate.so)
 
-set(LIB_PATH /usr/lib/x86_64-linux-gnu)
+#set(LIB_PATH /usr/lib/x86_64-linux-gnu)
+set(LIB_PATH /usr/lib/aarch64-linux-gnu)
```

Comment:
is this something for the dev board?

Reply:
yes, will probably be changed to depend on an external argument we provide to the Makefile to define what platform we're dealing with before building the backend

Reply:
yep, then we should handle this via an arg. Will set it back to the normal docker one for now then